src/hack/azext_hack/custom.py

```diff
@@ -0,0 +1,53 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# # --------------------------------------------------------------------------------------------
+from uuid import uuid4
+from knack.log import get_logger
+
+from .utils import (
+    create_resource_group,
+    create_database,
+    create_website,
+    set_website_settings,
+    create_cogsvcs_key
+)
+
+logger = get_logger(__name__)
+
+
+def hack_up(cmd, name, runtime, database, ai=None):
+    location = 'westus'
```

Comment:
we could have default location, but it would be nice to be able to change it. it may happen for some reason that particular resource is unavailable lin "westus" location.